app/src/main/java/com/codesoom/assignment/domain/Task.java

```diff
@@ -0,0 +1,51 @@
+package com.codesoom.assignment.domain;
+
+import java.util.Objects;
+
+public class Task {
+    private Long id;
+    private String title;
+
+    public Task() {
+    }
+
+    public Task(String title) {
+        this.title = title;
+    }
+
+    public Task(Long id, String title) {
+        this.id = id;
+        this.title = title;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void updateTitle(String updated) {
+        if (!title.equals(updated)) {
+            title = updated;
+        }
```

Comment:
그냥 이렇게 해도 똑같을 것 같아요. `equals` 비교가 얼마나 유의미한지도 생각해 볼만 하겠군요.
```suggestion
            title = updated;
```

Reply:
생각해보니 매번 값을 비교하는 과정이 들어가면 더 느려질 수 있겠네요. 👍